src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -41,6 +43,6 @@ protected static function createClient(array $options = [], array $server = [])
 
         $client->setServerParameters($server);
 
-        return $client;
+        return static::$client = $client;
```

Comment:
if you store the client in the property, you must ensure you clean it on teardown so that we don't keep unwanted references to it.

Reply:
Yep, done! Thanks :+1: